{{cookiecutter.project_slug}}/compose/local/docs/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN apt-get update \
+    # dependencies for building Python packages
+    && apt-get install -y build-essential \
+    # psycopg2 dependencies
+    && apt-get install -y libpq-dev \
+    # Translations dependencies
+    && apt-get install -y gettext \
+    # Enable Sphinx output to latex and pdf
+    && apt-get install -y texlive-latex-recommended \
```

Comment:
Most people probably don't care about latex so I could take this out.

Reply:
Depends how heavy this is I suppose, I don't mind either way.

Reply:
texlive-latex-recommended should be at least >200MBs if not half a gb:

```sh
$ docker run --rm -it python:3.8-slim-buster bash
# apt-get update && apt-get install --dry-run texlive-latex-recommend
...
The following NEW packages will be installed:
  cpp cpp-8 dbus fontconfig-config fonts-dejavu-core fonts-droid-fallback fonts-lmodern fonts-noto-mono krb5-locales libapparmor1 libauthen-sasl-perl libavahi-client3
  libavahi-common-data libavahi-common3 libbrotli1 libbsd0 libcairo2 libcups2 libcupsfilters1 libcupsimage2 libdata-dump-perl libdbus-1-3 libdrm-amdgpu1 libdrm-common
  libdrm-intel1 libdrm-nouveau2 libdrm-radeon1 libdrm2 libedit2 libelf1 libencode-locale-perl libfile-basedir-perl libfile-desktopentry-perl libfile-listing-perl
  libfile-mimeinfo-perl libfont-afm-perl libfontconfig1 libfontenc1 libfreetype6 libgdbm-compat4 libgl1 libgl1-mesa-dri libglapi-mesa libglib2.0-0 libglib2.0-data libglvnd0
  libglx-mesa0 libglx0 libgraphite2-3 libgs9 libgs9-common libgssapi-krb5-2 libharfbuzz-icu0 libharfbuzz0b libhtml-form-perl libhtml-format-perl libhtml-parser-perl
  libhtml-tagset-perl libhtml-tree-perl libhttp-cookies-perl libhttp-daemon-perl libhttp-date-perl libhttp-message-perl libhttp-negotiate-perl libice6 libicu63 libidn11
  libijs-0.35 libio-html-perl libio-socket-ssl-perl libio-stringy-perl libipc-system-simple-perl libisl19 libjbig0 libjbig2dec0 libjpeg62-turbo libk5crypto3 libkeyutils1
  libkpathsea6 libkrb5-3 libkrb5support0 liblcms2-2 libldap-2.4-2 libldap-common libllvm7 liblwp-mediatypes-perl liblwp-protocol-https-perl libmailtools-perl libmpc3 libmpfr6
  libnet-dbus-perl libnet-http-perl libnet-smtp-ssl-perl libnet-ssleay-perl libopenjp2-7 libpaper-utils libpaper1 libpciaccess0 libperl5.28 libpixman-1-0 libpng16-16
  libpotrace0 libptexenc1 libsasl2-2 libsasl2-modules libsasl2-modules-db libsensors-config libsensors5 libsm6 libsynctex2 libteckit0 libtexlua52 libtexlua53 libtexluajit2
  libtext-iconv-perl libtie-ixhash-perl libtiff5 libtimedate-perl libtry-tiny-perl liburi-perl libwebp6 libwoff1 libwww-perl libwww-robotrules-perl libx11-6 libx11-data
  libx11-protocol-perl libx11-xcb1 libxau6 libxaw7 libxcb-dri2-0 libxcb-dri3-0 libxcb-glx0 libxcb-present0 libxcb-render0 libxcb-shape0 libxcb-shm0 libxcb-sync1 libxcb1
  libxcomposite1 libxcursor1 libxdamage1 libxdmcp6 libxext6 libxfixes3 libxft2 libxi6 libxinerama1 libxml-parser-perl libxml-twig-perl libxml-xpathengine-perl libxml2 libxmu6
  libxmuu1 libxpm4 libxrandr2 libxrender1 libxshmfence1 libxt6 libxtst6 libxv1 libxxf86dga1 libxxf86vm1 libxxhash0 libzzip-0-13 lmodern lsb-base perl perl-modules-5.28
  perl-openssl-defaults poppler-data sensible-utils shared-mime-info t1utils tex-common texlive-base texlive-binaries texlive-latex-base texlive-latex-recommended ucf
  x11-common x11-utils x11-xserver-utils xdg-user-dirs xdg-utils xfonts-encodings xfonts-utils
0 upgraded, 187 newly installed, 0 to remove and 0 not upgraded.
```

I think we should take it out if we can.

Reply:
That's fine, I'll remove.